### PR TITLE
Build check fixes

### PR DESCRIPTION
**Pull request by daschuer**
This aims to fix all pending clang-tidy and clazy warnings.

### CODE REVIEW

**Review by Swiftb0y — APPROVED** [review]
LGTM. waiting for CI

File: src/widget/weffectchainpresetselector.cpp [hunk 1]
```diff
@@ -37,7 +37,7 @@ void WEffectChainPresetSelector::setup(const QDomNode& node, const SkinContext&
     }
 
     auto chainPresetListUpdateSignal = &EffectChainPresetManager::effectChainPresetListUpdated;
-    auto pQuickEffectChain = qobject_cast<QuickEffectChain*>(m_pChain.data());
+    auto* pQuickEffectChain = qobject_cast<QuickEffectChain*>(m_pChain.data());
     if (pQuickEffectChain) {
```
Comment (JoergAtGithub) [inline src/widget/weffectchainpresetselector.cpp:40]: Is it clean to use `if (pQuickEffectChain) {` or should it better be `if (pQuickEffectChain!=nullptr) {` ?

Comment (daschuer) [reply src/widget/weffectchainpresetselector.cpp:40]: Both works equal good. We have no preference for one or the other. I personal prefer the implicit bool cast.

Comment (JoergAtGithub) [reply src/widget/weffectchainpresetselector.cpp:40]: Ok

**Comment by JoergAtGithub** [comment]
LGTM! Thank you!

**Pull request merged**